mods/mapgen/deco.lua

```diff
@@ -199,70 +162,38 @@ for i in ipairs(decoration_list) do
-	local kubwakubwa_eggs_land = minetest.get_decoration_id("animals:kubwakubwa_eggs_land")
-	local darkasthaan_eggs     = minetest.get_decoration_id("animals:darkasthaan_eggs")
-	local pegasun_eggs         = minetest.get_decoration_id("animals:pegasun_eggs")
-	local sneachan_eggs        = minetest.get_decoration_id("animals:sneachan_eggs")
-minetest.set_gen_notify(
-	{decoration = true},
-	{
-		gundu_eggs,
-		sarkamos_eggs,
-		impethu_eggs,
-		kubwakubwa_eggs,
-		kubwakubwa_eggs_land,
-		darkasthaan_eggs,
-		pegasun_eggs,
-		sneachan_eggs
+local egg_names = {
+	"gundu_eggs",
+	"sarkamos_eggs",
+	"impethu_eggs",
+	"kubwakubwa_eggs",
+	"kubwakubwa_eggs_land",
+	"darkasthaan_eggs",
+	"pegasun_eggs",
+	"sneachan_eggs",
 	}
-	)
+local eggs_nearby = {}
+for i in ipairs(egg_names) do -- get decoration IDs
+	table.insert(eggs_nearby, minetest.get_decoration_id("animals:"..egg_names[i]))
+	end
+minetest.set_gen_notify({decoration = true}, eggs_nearby)
 minetest.register_on_generated(
 	function(minp, maxp, blockseed) -- start nodetimers
 		local gennotify = minetest.get_mapgen_object("gennotify")
 		local poslist = {}
-		for _, pos in ipairs(gennotify["decoration#"..gundu_eggs] or {}) do
-			local gundu_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, gundu_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..sarkamos_eggs] or {}) do
-			local sarkamos_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, sarkamos_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..impethu_eggs] or {}) do
-			local impethu_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, impethu_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..kubwakubwa_eggs] or {}) do
-			local kubwakubwa_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, kubwakubwa_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..kubwakubwa_eggs_land] or {}) do
-			local kubwakubwa_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, kubwakubwa_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..darkasthaan_eggs] or {}) do
-			local darkasthaan_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, darkasthaan_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..pegasun_eggs] or {}) do
-			local pegasun_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, pegasun_eggs_pos)
-			end
-		for _, pos in ipairs(gennotify["decoration#"..sneachan_eggs] or {}) do
-			local sneachan_eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
-			table.insert(poslist, sneachan_eggs_pos)
+		for i in ipairs(egg_names) do
+			for j, pos in ipairs(gennotify["decoration#"..eggs_nearby[i]] or {}) do
+				local eggs_pos = {x = pos.x, y = pos.y + 1, z = pos.z}
+				table.insert(poslist, eggs_pos)
+				minetest.chat_send_all("woot")
```

Comment:
I forgot to delete this line before committing.

Reply:
Make the change in your branch, commit, and push to github, it should show up automatically.